### PR TITLE
Header and footer menus; dynamically retrieve article media; lists; new line EOF

### PR DESCRIPTION
**Pull request by ljubar**

### CODE REVIEW

**Review by ljubar — COMMENTED** [review]
@ahilles107 can you review whole PR please - we need it merged so Maja can sync and continue work on it

File: amp/amp-theme/partials/metadata.html.twig [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "datePublished": "{{ object.publishedAt|date('Y-m-d H:i:s') }}",
+        "dateModified": "{{ object.updatedAt|date('Y-m-d H:i:s') }}",
+        "description": "{{ object.lead }}",
+        "author": {
+          "@type": "Person",
+          "name": "{{ object.metadata.byline }}"
+        },
+        "publisher": {
+          "@type": "Organization",
+          "logo": {
```
Comment (ahilles107) [inline amp/amp-theme/partials/metadata.html.twig]: it's better to comment it out than providing github logo

File: amp/amp-theme/partials/metadata.html.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "@type": "ImageObject",
+            "url": "https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png",
+            "width": 125,
+            "height": 60
+          },
+          "name": "{{ tenant.organization.name }}"
+        },
+        "image": {
+          "@type": "ImageObject",
+          "url": "http://cdn.ampproject.org/leader.jpg",
```
Comment (ahilles107) [inline amp/amp-theme/partials/metadata.html.twig]: there is 404 on that url

File: public/css/elements/navigation.sass [hunk 3]
```diff
@@ -8,9 +8,12 @@
...
\ No newline at end of file
+     font-size: 2rem
+     font-weight: 900
+     display: inline-block
+     margin-right: 40px
+     &.current 
+       a
+         color: $colorTheme
+       a:hover
+         color: $colorTheme
```
Comment (ahilles107) [inline public/css/elements/navigation.sass]: empty line missing

File: public/img/hamburger.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg enable-background="new 0 0 32 32" height="32px" id="Слой_1" version="1.1" viewBox="0 0 32 32" width="32px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g id="Menu"><path d="M1,7h30c0.552,0,1-0.448,1-1c0-0.552-0.448-1-1-1H1C0.448,5,0,5.448,0,6C0,6.552,0.448,7,1,7z" fill="#121313"/><path d="M31,15H1c-0.552,0-1,0.448-1,1c0,0.552,0.448,1,1,1h30c0.552,0,1-0.448,1-1C32,15.448,31.552,15,31,15z" fill="#121313"/><path d="M31,25H1c-0.552,0-1,0.448-1,1c0,0.552,0.448,1,1,1h30c0.552,0,1-0.448,1-1C32,25.448,31.552,25,31,25z" fill="#121313"/></g><g/><g/><g/><g/><g/><g/></svg>
```
Comment (ahilles107) [inline public/img/hamburger.svg]: empty line missing

File: views/article.html.twig [hunk 5]
```diff
@@ -34,38 +34,80 @@
...
 						<div class="article__entrefilet article__floatLeft">
 							<h3>Entrefilet</h3>
 							<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed eu enim in velit bibendum gravida nec et odtio. Cras congue, orci vitae eleifend congue, felis mi tincidunt risus, et lobortis turpis ex vel velit. Nullam pellentesque tempor magna a fermentum.</p>
 						</div>
+						#}
 					</div>
 
+					<div id="disqus_thread"></div>
+<script>
+// var disqus_config = function () {
```
Comment (ahilles107) [inline views/article.html.twig]: let's remove commented code here

**Pull request merged**